app-laptop/i8kutils/i8kutils-1.42.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Dell Inspiron and Latitude utilities"
+HOMEPAGE="https://launchpad.net/i8kutils"
+SRC_URI="https://launchpad.net/i8kutils/trunk/${PV}/+download/${P/-/_}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples tk"
+
+DEPEND="tk? ( dev-lang/tk:0 )"
+RDEPEND="${DEPEND}
+         sys-power/acpi"
+
+S="${WORKDIR}/${PN}"
+
+DOCS=( README.i8kutils )
+
+src_prepare() {
+	sed \
+		-e '/^CC/d' \
+		-e '/^CFLAGS/d' \
+		-e 's: -g : $(LDFLAGS) :g' \
+		-i Makefile || die
+
+	tc-export CC
+}
+
+src_install() {
+	dobin i8kctl i8kfan
+	doman i8kctl.1
+
+	use examples && dodoc -r examples
+
+	newinitd "${S}"/debian/i8kutils.i8kmon.init i8k
```

Comment:
So have you fully runtested this debian patch in the gentoo system? Including runtime?